### PR TITLE
Update sitemap.xml

### LINKED ISSUE #415: Feat: sitemap addition for website
### Is there an existing issue raised for this?

- [X] I have searched the existing issues

### Describe the feature

Yes, the feature request is related to a problem. The problem is that the current website lacks a sitemap. Without a sitemap, it becomes challenging for search engines and users to navigate and discover all the pages and content on the site. This can result in lower search engine rankings, decreased visibility, and a poor user experience.

### Add Screenshots

stich

In the attached screenshot, I have highlighted the website name with the expected presence of a sitemap.xml file. However, upon inspection, it is evident that the sitemap is not being displayed. This screenshot underscores the problem of the current website lacking a visible sitemap, further emphasizing the need for its implementation.

### Record Options

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'23 contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by NikhilRaikwar**
# Description

This commit includes updates to the sitemap.xml file. The updates reflect changes in the website's content and structure. By updating the sitemap.xml file, we ensure that it accurately represents the URLs and metadata of the website's pages.

Fixes #415 

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

# Screenshots

## Before
Screenshot 2023-06-28 171324

## After
code

**Include video if needed**

### CODE REVIEW

**Comment by UBA-GCOEN** [comment]
Hello @NikhilRaikwar!
Thank you for submitting this pull request Your pull request will be reviewed by maintainers shortly. Meanwhile make sure you have referenced the respected issue in this pr Your pr will not be merged if it does not have reference to respected issue
please read our Contributing Guidelines https://github.com/UBA-GCOEN/StichHub/blob/main/CONTRIBUTING.md And Code of Conduct https://github.com/UBA-GCOEN/StichHub/blob/main/CODE_OF_CONDUCT.md

**Review by Siddhant-Patil0203 — COMMENTED** [review]
@NikhilRaikwar  Looks good 🚀

**Review by Siddhant-Patil0203 — APPROVED** [review]

**Comment by NikhilRaikwar** [comment]
Hey @Siddhant-Patil0203 Pls also add the label of gssoc23 or level of issue in the pr after merging.

**Pull request merged**
